data/data/bootstrap/files/usr/local/bin/release-image-download.sh.template

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env bash
+set -euo pipefail
```

Comment:
```
# This script continues trying to download the release image until
# successful because we cannot use Restart=on-failure with a oneshot
# service: https://github.com/systemd/systemd/issues/2582.
```

This will make this script fail and exit on first error.. see the reason above for why that can't happen.. this script needs to keep retrying internally until this succeeds..

am i wrong?

Reply:
Also the when using `Require` see https://github.com/systemd/systemd/issues/1312

We need to make sure that the bootstrap-host units are resilient to transient pull-image or other errors. Keep retying until timeout no matter what.

Reply:
Oh wow, I didn't know about that systemd issue.  That said, I think we're still good here because `set -e` doesn't cause the script to exit when a failing command is part of a conditional; IOW, the code should still work.

That said, it also doesn't *currently* really do much in this script either...so we could remove it.  But, it is dangerous I think to not have it because people may copy/paste scripts and I think we really, really want all shell script to be in "strict mode".

(I mean, really we want to not use shell script at all since it's full of [horrifying traps](https://www.oilshell.org/blog/2018/01/28.html) but...)